CMakeLists.txt

```diff
@@ -117,6 +115,37 @@ target_include_directories(musica
 
 add_subdirectory(src)
 
+#
+# jiwon 7/21 - in progress
+#
+####################
+# MUSICA - fortran
+add_library(musica-fortran STATIC src/fortran_interface/micm/interface.F90)
+add_library(musica::musica-fortran ALIAS musica-fortran)
+
+set_target_properties(musica-fortran PROPERTIES
+  ARCHIVE_OUTPUT_DIRECTORY ${MUSICA_LIB_DIR}
+  Fortran_MODULE_DIRECTORY ${MUSICA_MOD_DIR}
+)
+
+target_link_libraries(musica-fortran 
+  PUBLIC
+    PkgConfig::netcdff
+    ${JSON_LIB}
+)
```

Comment:
I don't think we'll need this since we aren't linking to json or netcdff for this little target
```suggestion
```